.github/ISSUE_TEMPLATE/proposal.yml

```diff
@@ -0,0 +1,26 @@
+name: Proposal
+description: Proposal for feature to include in Interop 2023
+labels: proposal
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to make a proposal for Interop 2023!
+
+      Please see the [Interop 2023 README](https://github.com/web-platform-tests/interop/blob/main/2023/README.md) for advice on how to make a proposal and what to expect from the process.
```

Comment:
Can we guide people more fully, either with this Markdown or longer descriptions of each field? Let's assume people won't follow a link to a readme.

Reply:
Good idea! I've inlined some things from the charter but with slight tweaks for the "voice" used. The rendered result of these descriptions is a bit too small IMHO, but if we put things into the textfield values instead it won't be possible to follow links, which seems bad too.

Please review!